substrate/primitives/application-crypto/src/ecdsa.rs

```diff
@@ -48,6 +49,18 @@ impl RuntimePublic for Public {
 		sp_io::crypto::ecdsa_verify(signature, msg.as_ref(), self)
 	}
 
+	fn generate_pop(&mut self, key_type: KeyTypeId) -> Option<Self::Signature> {
+		let pub_key_as_bytes = self.to_raw_vec();
+		let pop_statement = [POP_CONTEXT_TAG, pub_key_as_bytes.as_slice()].concat();
+		sp_io::crypto::ecdsa_sign(key_type, self, pop_statement.as_slice())
+	}
+
+	fn verify_pop(&self, pop: &Self::Signature) -> bool {
```

Comment:
because they haven't use AppPair::verify for signature verification, perhaps we follow the same path and call ecdsa_verify here.